app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -273,6 +273,10 @@ class EditPatientScreen(context: Context, attributeSet: AttributeSet) : Relative
     deletePatient.setOnClickListener { router.push(DeletePatientScreenKey(screenKey.patient.uuid).wrap()) }
```

Comment:
The commit message says `Fetch the colonyOrVillage list when the EditPatientScreen is created`, but here we are making changes to the ViewRenderer, can you update the commit message to reflect that, please?

Reply:
We are technical not rendering it when screen is created, we don't have to fix this. But just an FYI